src/index.html

```diff
@@ -1,14 +1,27 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="page">
+    <section class="page__container container">
+      <div class="container__clock clock">
+        <div class="clock__hours-line hours-line hours-line--one"></div>
+        <div class="clock__hours-line hours-line hours-line--two"></div>
+        <div class="clock__hours-line hours-line hours-line--three"></div>
+        <div class="clock__hours-line hours-line hours-line--four"></div>
+
+        <div class="clock__inner-block inner-block">
+          <div class="inner-block__arrow arrow arrow--hour"></div>
+          <div class="inner-block__arrow arrow arrow--min"></div>
+          <div class="inner-block__arrow arrow arrow--sec"></div>
+        </div>
+      </div>
```

Comment:
```suggestion
      <div class="container__clock clock">
        <div class="clock__line clock__line--one"></div>
        <div class="clock__line clock__line--two"></div>
        <div class="clock__line clock__line--three"></div>
        <div class="clock__line clock__line--four"></div>

        <div class="clock__hands">
          <div class="clock__hand clock__hand--hour"></div>
          <div class="clock__hand clock__hand--min"></div>
          <div class="clock__hand clock__hand--sec"></div>
        </div>
      </div>
```

You don't need to have separate blocks for lines and hands as they are parts of the clock.
Also, you can make BEM more clear and simpler.

Reply:
fixed